# HG changeset patch
# User Matti Picus <matti.picus@gmail.com>
# Date 1692026252 -10800
#      Mon Aug 14 18:17:32 2023 +0300
# Branch conda-3.9-v7.3.12
# Node ID 56b6319f7d74c233348a2109a3de883d5d38a581
# Parent  9563129b6f7d3d60334f3a5d42a8e9a74f5f31da
do not create c-extensions when running tests

diff -r 9563129b6f7d -r 56b6319f7d74 lib-python/3/test/support/__init__.py
--- a/lib-python/3/test/support/__init__.py	Sat Aug 12 23:12:09 2023 +0300
+++ b/lib-python/3/test/support/__init__.py	Mon Aug 14 18:17:32 2023 +0300
@@ -28,11 +28,6 @@
 
 from .testresult import get_test_runner
 
-# This will create the _testmultiphaseimport c-extension module if it does not exist
-import _testmultiphase_build
-# This will create the _ctypes_test c-extension module if it does not exist
-import _ctypes_test_build
-
 __all__ = [
     # globals
     "PIPE_MAX_SIZE", "verbose", "max_memuse", "use_resources", "failfast",
